data/sql/updates/pending_db_world/rev_1554339238034040500.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1554339238034040500');
+
+INSERT INTO `creature_text` VALUES (22252, 0, 0, 'Me so hungry! YUM!', 12, 0, 100, 0, 0, 0, 0, 0, 'Dragonmaw Peon Say1');
```

Comment:
`DELETE` before `INSERT` please.
Also, insert all the values with one single query (INSERT INTO a VALUES (),(),();)